cookbooks/bcpc-hadoop/recipes/disks.rb

```diff
@@ -31,7 +31,17 @@
       action [:enable, :mount]
     end
   end
-  node.set[:bcpc][:hadoop][:mounts] = (0..node[:bcpc][:hadoop][:disks].length-1).to_a
+  if node.roles.include? "BCPC-Hadoop-Head" and node[:bcpc][:hadoop][:head_disks] != nil then
```

Comment:
We do not run graphite on Kafka, at least not that I see in current roles definitions for Server and Zookeeper